timeseries/setup.py

```diff
@@ -32,10 +32,8 @@
     "accelerate",  # version range defined in `core/_setup_utils.py`
     "gluonts>=0.14.0,<0.14.4",  # 0.14.4 caps pandas<2.2
     "networkx",  # version range defined in `core/_setup_utils.py`
-    # TODO: update statsforecast to v1.5.0 - resolve antlr4-python3-runtime dependency clash with multimodal
-    "statsforecast>=1.4.0,<1.5",
+    "statsforecast>=1.7.0,<1.8",
```

Comment:
We also need to pin `fugue>=0.9.0` for this to work. Older `fugue` versions resulted in the nasty dependency clash with `omegaconf` (https://github.com/fugue-project/fugue/issues/509#issuecomment-1972877209)